cilium/CFP-30984-dns-proxy-ha.md

```diff
@@ -85,67 +87,114 @@ message Result {
 }
 ```
 
-Method : GetEndpointMetadataByIP
```

Comment:
It might be better if we present both the options since we aren't sure of which path to take right now. Fits perfectly with the `Impacts / Key Questions` section.

Reply:
Agreed. I added all the three options in the other PR and removed it from the proposed one as we want to rely on bpf maps.